.cirrus.yml

```diff
@@ -961,17 +972,14 @@ farm_test_task:
 buildah_bud_test_task:
     name: *std_name_fmt
     alias: buildah_bud_test
-    # Docs: ./contrib/cirrus/CIModes.md
-    only_if: *not_tag_magic
-    # skip when: - it is a PR (we never want to skip on nightly tests); and
-    #            - CI:ALL title is not set; and
-    #            - no danger files are changed; and
-    #            - no build source files are changed and no bud tests
-    skip: >-
-        $CIRRUS_PR != '' &&
-        $CIRRUS_CHANGE_TITLE !=~ '.*CI:ALL.*' &&
-        !changesInclude('.cirrus.yml', 'Makefile', 'contrib/cirrus/**', 'vendor/**', 'hack/**', 'version/rawversion/*') &&
-        !changesInclude('**/*build*.go', 'test/buildah-bud/**')
+    # Docs: ./contrib/cirrus/CIModes.md (Cirrus Task contexts and runtime modes)
+    # only when: - main rules (see doc above); or
+    #            - build source, bud tests files are changed
```

Comment:
"build source" does not fully communicate the sense of "source code related to the podman build subcommand". But that's too long. Any ideas for a happy medium?

Reply:
used "podman build source code files", also added a comment here that a buildah vendor is already covered by the main rules as this might be something a reader thinks about